doc/features/build-arm-images.md

```diff
@@ -0,0 +1,210 @@
+    ```
+    git apply ./arm64-aci.patch
+    ```
+4. **Build ACI Containers binaries**
+    ```
+    make clean-dist-static all-static
+    ``` 
+5. **Copy Iptables binaries**
+
+    Once the binaries have been built we need to copy some additional files to the dist-static directory in aci-containers. We will use `docker/copy_iptables.sh` script to copy iptables binaries and libraries from opflex-build-base container built in the arlier step into dist-static to be used by consumers of iptables like the host agent container.
+    ```
+    ./docker/copy_iptables.sh $IMAGE_BUILD_REGISTRY/opflex-build-base:$IMAGE_BUILD_TAG dist-static
+    ```   
+
+### Build ARM-specific OpenvSwitch:
+ 
+We will build OpenvSwitch with the script [openvswitch-arm64.sh](openvswitch-arm64.sh). Copy the script to aci-containers directory and run it. 
+ 
+Script usage:
+```
+./build_openvswitch.sh  $IMAGE_BUILD_REGISTRY $IMAGE_BUILD_TAG
+```
+This script can take three parameters:
+1. Image Registry
+2. Image Tag
+3. Build args: We can provide additional build arguments for docker buildx build commands, for example if you are using an http proxy you may use the script as:
+`./build_openvswitch.sh $IMAGE_BUILD_REGISTRY $IMAGE_BUILD_TAG "--build-arg https_proxy=${HTTP_PROXY} --build-arg no_proxy=${NO_PROXY}"`
+
+The above steps will build two containers:
+```
+1. $IMAGE_BUILD_REGISTRY/openvswitch-base:$IMAGE_BUILD_TAG
+2. $IMAGE_BUILD_REGISTRY/openvswitch:$IMAGE_BUILD_TAG
+```
+
+###  Build ARM-specific ACI-CNI Container Images 
+We will use Docker Buildx to create multi-architecture images: `docker buildx build --no-cache -t <tag>  --platform linux/arm64 -f Dockerfile --push .`
+
+**1. Build the Containers:**
+
+```
+docker buildx build --no-cache --platform linux/arm64 -t ${IMAGE_BUILD_REGISTRY}/aci-containers-controller:${IMAGE_BUILD_TAG} --file=docker/travis/Dockerfile-controller .
+docker buildx build --no-cache --platform linux/arm64 --target without-ovscni -t ${IMAGE_BUILD_REGISTRY}/aci-containers-host:${IMAGE_BUILD_TAG} --file=docker/travis/Dockerfile-host .
+docker buildx build --no-cache --platform linux/arm64 -t ${IMAGE_BUILD_REGISTRY}/aci-containers-operator:${IMAGE_BUILD_TAG} --file=docker/travis/Dockerfile-operator .
+```
+
+**2. Verify the Images:**
+
+Run the following command to check the images:
+```
+docker images
+REPOSITORY                               TAG                       IMAGE ID       CREATED         SIZE
+my_registry/aci-containers-operator      v0.1-test                 2bccbe38a7fd   7 hours ago     589MB
+my_registry/aci-containers-controller    v0.1-test                 aa8815e53b15   7 hours ago     638MB
+my_registry/aci-containers-host          v0.1-test                 1a2300555aee   7 hours ago     727MB
+my_registry/openvswitch                  v0.1-test                 1365a40ef73f   7 hours ago     681MB
+my_registry/openvswitch-base             v0.1-test                 8f36e0df45cf   7 hours ago     1.33GB
+```
```

Comment:
we need to include the opflex and acc-provision-operator images in this list

Reply:
included opflex, will include acc-provision-operator in the follow up PR.

Reply:
added info to build acc-provision-operator as well.